certified-connectors/SmartCOMM Doc-Gen ap10-sb/Readme.md

```diff
@@ -0,0 +1,113 @@
+# SmartCOMM On Demand Generation Connector
+Smart Communications™ is a leading cloud-based platform for enterprise customer communications. As the only cloud solution ranked as a Leader in Gartner’s Magic Quadrant for Customer Communications Management, more than 350 global brands – many in the world’s most highly regulated industries – rely on Smart Communications to make multi-channel customer communications more meaningful, while also helping them simplify their processes and operate more efficiently.
```

Comment:
@tajujsag Our legal team will not let us certify with either any copyright or registered trademarks anywhere in the connector artifacts. So, please remove them from all the connector artifacts.

Reply:
I removed ™ character from the readme text. The other occurences is apeDefinition.swagger.json are part of the parameter description, simply telling the fact, that they are valid characters in tags. Those cannot be removed, otherwise the parameter description would be incorrect.